Quickstarts/read-d2c-messages/pom.xml

```diff
@@ -0,0 +1,40 @@
+<!-- Copyright (c) Microsoft. All rights reserved. --><!-- Licensed under the MIT license. See LICENSE file in the project root for full license information. -->
+<project>
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>com.microsoft.docs.iothub.samples</groupId>
```

Comment:
Can update this to match the other samples' groupId

Reply:
this is weird too, it should not be the case any more.